surround/visualise/visualise_classifier.py

```diff
@@ -174,6 +171,73 @@ def visualise(self, surround_data, config):
 
         print("============[End of visualisation]===================")
 
+def classification_report(y_true, y_pred, classes):
+    results = {}
+
+    for name in classes:
+        tp = len([(yt, yp) for yt, yp in zip(y_true, y_pred) if yt == yp and yt == name])
+        fp = len([(yt, yp) for yt, yp in zip(y_true, y_pred) if yt != yp and yp == name])
+        fn = len([(yt, yp) for yt, yp in zip(y_true, y_pred) if yt != yp and yt == name])
+
+        precision = tp / (tp + fp)
+        recall = tp / (tp + fn)
```

Comment:
Getting division by zero error at this line. To recreate run:
surround viz -g ground -p predict inputdata.csv
With the attached csv
[inputdata.csv.zip](https://github.com/a2i2/surround/files/3843889/inputdata.csv.zip)